simplq/src/store/allQueues/index.js

```diff
@@ -0,0 +1,5 @@
+import reducer, { selectQueues } from './allQueuesSlice';
+
```

Comment:
Thought renaming it to allQueueSlices would make it more apparent that this refers to the list of queues we see on the homepage.

Reply:
I would rather call it userQueues and selectUserQueues, implying the current user.